### ISSUE TITLE
Table 'mailcoach.media' doesn't exist

### ISSUE THREAD

**Issue by rnleal** [spatie/mailcoach-support#41]
Hi Spatie Team,

When importing subscribers, I get this sql error

`SQLSTATE[42S02]: Base table or view not found: 1146 Table 'mailcoach.media' doesn't exist (SQL: select * from `media` where `media`.`model_id` = 3 and `media`.`model_id` is not null and `media`.`model_type` = Spatie\Mailcoach\Models\SubscriberImport) (View: /home/vagrant/code/mailcoach/vendor/spatie/laravel-mailcoach/resources/views/app/emailLists/importSubscribers.blade.php)`

Running `php artisan migrate` doesn't solve it. Please advise. 

and thank you for your works!

**Comment by rnleal** [comment]
update: installing [spatie/laravel-medialibrary](https://github.com/spatie/laravel-medialibrary)  solves this issue. 

Not sure if this is the right fix though?

**Comment by riasvdv** [comment]
This is indeed an issue, for imports you need to publish the migration of the laravel-medialibrary package.

```
php artisan vendor:publish --provider="Spatie\MediaLibrary\MediaLibraryServiceProvider" --tag="migrations"
```

I'll fix this in a new release so the Mailcoach service provider also publishes the medialibrary migrations

**Comment by riasvdv** [comment]
Fixed in 1.5.1

**Comment by ziming** [comment]
Hi @riasvdv 

I'm using laravel media library in my project before mailcoach for files uploads. What happens now? Another media migration is published.

Will this conflict with my existing media table?
